baseapp/baseapp.go

```diff
@@ -933,3 +953,19 @@ func (app *BaseApp) DefaultProcessProposal() sdk.ProcessProposalHandler {
 		return abci.ResponseProcessProposal{Status: abci.ResponseProcessProposal_ACCEPT}
 	}
 }
+
+// NoOpPrepareProposal defines a no-op PrepareProposal handler. It will always
+// return the transactions sent by the client's request.
+func NoOpPrepareProposal() sdk.PrepareProposalHandler {
```

Comment:
Is this still needed?

Reply:
Yes, I'd prefer this to be the default -- currently, it's not.